src/Clients/FlagClient.cs

```diff
@@ -0,0 +1,45 @@
+using System.Collections.Generic;
+using System.Net;
+using System.Threading.Tasks;
+using StreamChat.Models;
+using StreamChat.Rest;
+
+namespace StreamChat.Clients
+{
+    public class FlagClient : ClientBase, IFlagClient
+    {
+        internal FlagClient(IRestClient client) : base(client)
+        {
+        }
+
+        public async Task<ApiResponse> FlagUserAsync(string flaggedId, string flaggerId)
+            => await PostFlagAsync(flaggedId, flaggerId, "flag", "user");
+
+        public async Task<ApiResponse> UnflagUserAsync(string flaggedId, string flaggerId)
```

Comment:
let's drop unflag since no action is done in server

Reply:
✅

Reply:
@ferhatelmas  how come you can flag message/user but you can't unflag them? Is there any other endpoint that covers that? Or is the design that this should be handled in chat dashboard?

Reply:
Unflag endpoints does nothing at the moment so adding is misleading (wasn't dropped not to break), there will be new moderation support on the way. Since this is a breaking release, good to drop them for now.